### ISSUE TITLE
add confirmation to export?

### ISSUE THREAD

**Issue by Droidem** [J-D-K/JKSM#24]
Is it possible to add a confirmation to export like import does?

**Comment by J-D-K** [comment]
Yes. It's possible. I've thought about it before, but I kept forgetting to add it.
